CMake/HPHPCompiler.cmake

```diff
@@ -122,14 +122,22 @@ if (${CMAKE_CXX_COMPILER_ID} STREQUAL "Clang" OR ${CMAKE_CXX_COMPILER_ID} STREQU
       "fno-canonical-system-headers"
       "Wvla"
     )
+
     list(APPEND RELEASE_CXX_OPTIONS
       "-param max-inline-insns-auto=100"
-      "-param early-inlining-insns=200"
-      "-param max-early-inliner-iterations=50"
       "-param=inline-unit-growth=200"
       "-param=large-unit-insns=10000"
     )
 
+    # The params bellow causes problem on GCC 4.8 4.8 and 5.4 on PPC64
+    # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=72855
+    if(NOT IS_PPC64)
+      list(APPEND RELEASE_CXX_OPTIONS
+        "-param early-inlining-insns=200"
+        "-param max-early-inliner-iterations=50"
+    )
```

Comment:
Minor nit: The closing parenthesis here needs to be indented one more level. (so that it's at the same level as `list(APPEND`.